tests/functional/reporting-non-auth.js

```diff
@@ -286,30 +268,23 @@ define(
           .end();
       },
 
-      "details param adds info to description": function() {
```

Comment:
I don't understand why this test is being changed... we still support the `details` param and need to test that it appends info to the textarea.